docs/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Flexy OSIsoft Changelog
 
+## v2.3
+### Major Changes
++ The connector now supports storing tags as different data types in OSIsoft with OMF
+### Minor Changes
++ Performance improvement when the connector is establishing an initial connection
++ The connector logs out when the data queue is behind
++ The tag indexing error is fixed
++ The webpage not saving new configurations is fixed
++ The connector now accounts for the new tag indexing that includes null points on tag gaps

```

Comment:
Generally changelog (as well as commit messages) should be concise and start with a verb.

Ex: Fixed tag indexing error [Rather than - The tag indexing error is fixed]